packages/plans-grid-next/src/hooks/data-store/use-grid-plans.tsx

```diff
@@ -210,6 +210,9 @@ export const usePlanTypesWithIntent = ( {
 		case 'plans-playground':
 			planTypes = [ TYPE_BUSINESS, TYPE_ECOMMERCE ];
 			break;
+		case 'plans-playground-premium':
```

Comment:
@zaerl I can also add a comment here and in types definition that this is currently not in use unless you think we should just remove it, and defer its addition.

Reply:
It's a good idea.